### DISCUSSION TITLE
Request to Join Gene Ontology Organization

### DISCUSSION THREAD

**Discussion by Emem-studio** [geneontology/helpdesk#508] [General]
Dear Gene Ontology Team,

My GitHub username is **Emem-studio**, and I would like to request access to the Gene Ontology organization. Please let me know if any additional details are required.

Best regards

**Comment by suzialeksander** [comment]
Hi @Emem-studio, 

Most of our data & products are available for anyone to read through our repos. Full permissions are reserved for those actively working on the GOC project. If you are currently in a GO-contributing group, please let me know (as well as which group) and we can get the onboarding process started. 

Thanks!
